src/dash-table/derived/paginator.ts

```diff
@@ -11,6 +11,13 @@ import {
 export interface IPaginator {
     loadNext(): void;
     loadPrevious(): void;
+    loadFirst(): void;
+    loadLast(): void;
+    lastPage: number | undefined;
+    goToPage(page: number): void;
+    disablePrevious(): boolean;
+    disableNext(): boolean;
+    disableLast(): boolean;
```

Comment:
Non-blocking. `disablePrevious` is of form `action-noun` which typically indicates a modification will be made. `hasPrevious` is better as you are retrieving the value of some calculated state.